packages/ee/shared/src/lib/managed-authn/managed-authn-requests.ts

```diff
@@ -0,0 +1,12 @@
+import { Type, Static } from '@sinclair/typebox'
+
+export const ManagedAuthnRequestBody = Type.Object({
```

Comment:
Why not ManagedAuth, or ManagedAuthentication?

Reply:
authn is the defacto abbreviation for authentication, authz is for authorization